device/globals/player.gd

```diff
@@ -317,6 +317,16 @@ func teleport_pos(x, y):
 	set_position(Vector2(x, y))
 	_update_terrain()
 
+func turn_to(deg):
+	if deg < 0 or deg > 360:
+		vm.report_errors("player", ["Invalid degree to turn to " + String(deg)])
```

Comment:
I don't think there's any overhead in using the `String()` constructor, but `str()` might be more idiomatic?

Reply:
Fixed.